core/src/validator.rs

```diff
@@ -144,6 +145,7 @@ const WAIT_FOR_SUPERMAJORITY_THRESHOLD_PERCENT: u64 = 80;
 pub enum BlockVerificationMethod {
     #[default]
     BlockstoreProcessor,
+    UnifiedScheduler,
```

Comment:
> I think because this is a hidden argument, it's probably fine to do that, but wanted to bring it up.

exactly. :)

> (also thanks, looking into this made me realize I didn't unhide the block-production-method argument 😆 )

:heart: